MAUI/Pull-to-Refresh/getting-started.md

```diff
@@ -15,8 +15,8 @@ This section provides a quick overview of how to get started with the .NET MAUI
 
  1. Create a new .NET MAUI application in Visual Studio.
  2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfPullToRefresh to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.PullToRefresh and install it.
- 3. Import the control namespace `Syncfusion.Maui.PullToRefresh` in XAML or C# code.
- 4. Initialize the `SfPullToRefresh` control.
+ 3. Import the control namespace [Syncfusion.Maui.PullToRefresh](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PullToRefresh.html) in XAML or C# code.
```

Comment:
Remove both links here and add it above topic.

Reply:
Removed links and updated in above topic